### ISSUE TITLE
Show the IP address when --force-login is applied in terminal-based communicators.

### ISSUE THREAD

**Issue by GijsTimmers** [GijsTimmers/kotnetcli#60]
Something like: 

```
Netlogin openen........... [ OK ]
Oud IP-adres opzoeken..... [   . WA.IT .   ]
Oud IP-adres opzoeken..... [ 10. 92.128.101]
Formulier invullen........ [ OK ]
Gegevens invullen ........ [ OK ]
Gegevens opsturen......... [ OK ]
Uitloggen................. [DONE]
```

or 

```
Netlogin openen........... [     OK      ]
Oud IP-adres opzoeken..... [    WAIT     ]
Oud IP-adres opzoeken..... [10.92.128.101]
Formulier invullen........ [     OK      ]
Gegevens invullen ........ [     OK      ]
Gegevens opsturen......... [     OK      ]
Uitloggen................. [    DONE     ]
```

The found IP address should be green. @jovanbulck: what do you think?

**Comment by jovanbulck** [comment]
Nice feature idd :+1: 

It uncovers another underlying problem with the design of the communicators though. The difference/mixing between login- and logout-only methods is becoming ugly. We should thing about 2 different interfaces:

![cd_communicator](https://cloud.githubusercontent.com/assets/2464627/5683887/ebd50328-982b-11e4-8ab3-a751bc78ef50.png)

**Comment by jovanbulck** [comment]
The order is important in multiple inheritance (e.g. http://stackoverflow.com/questions/21657822/python-and-order-of-methods-in-multiple-inheritance):

`class ColoramaLoginCommunicator(ColoramaCommunicator, PlainTextLoginCommunicator)` will get its common methods from Colorama and its login related methods from PlainTextLogin...

**Comment by jovanbulck** [comment]
Another proposal:

I would stick with the feature freeze for v2.0.0 here and label this for milestone 3.0.0.

The cool thing then would be that once the software design for v1.0.0 is finished, implementing this feature could act as a *stress test* for the new design :-) It would be a good test as it involves distributed changes focused on a specific action (logging out and/or force login). Indeed, it requires changes to the different communicators, as well as the parsing code as well as the worker.

I therefore suggest, deferring this idea until the software design is finished (v2.0.0) and then try implementing this regarding the design. If it works, we have a nice proof-of-concept for the new design. If not we found another way for improving the design :p

**Comment by GijsTimmers** [comment]
Could you expand a bit on the diagram? The way I see it, `LoginQuietCommunicator` on the right should be `LogoutQuietCommunicator`, shouldn't it?

**Comment by jovanbulck** [comment]
> LoginQuietCommunicator on the right should be LogoutQuietCommunicator, shouldn't it?

yep indeed, good catch, I'll change this in the final diagram...

**Comment by jovanbulck** [comment]
I created a diagram depicting the ideas we discussed on multiple inheritance. Check it out in the communicator-readme: https://github.com/GijsTimmers/kotnetcli/tree/dev/communicator

**Comment by GijsTimmers** [comment]
Something like: 

```bash
$ kotnetcli.py
ik haal de credentials uit de keyring
ik wil vrolijke kleuren
ik wil inloggen
Netlogin openen............. [ OK ]
KU Leuven kiezen............ [ OK ]
Gegevens invoeren........... [ OK ]
Gegevens opsturen........... [ OK ]
Download:       [===========][100%]
Upload:         [===========][100%]
Huidige sessie: [10.92.35.28]
```


```bash
$ kotnetcli.py --get-sessions
Actieve sessies: [10.92.35.28]
                 [10.92.38.41]
                 [10.92.30.10]
```
